GVRf/Framework/framework/src/main/java/org/gearvrf/x3d/X3Dobject.java

```diff
@@ -3273,7 +3305,15 @@ else if (qName.equalsIgnoreCase("Appearance")) {
             } else if (qName.equalsIgnoreCase("ProximitySensor")) {
                 currentSensor = null;
             } else if (qName.equalsIgnoreCase("Text")) {
-                ;
+                // since the text node is complete, now calculate
```

Comment:
Looks like this whole block belongs to GVRTextViewSO. Something like updateTextPlaneSize() which doesn't even need arguments from what I can tell.